deploy/local_single_user_deploy.sh

```diff
@@ -208,28 +316,37 @@ stop() {
     echo -e "${DEBUG}All processes stopped${NC}"
 }
 
-
-
 # Usage function with new options
 usage() {
     echo "Usage: $0 [options]"
     echo "Options:"
-    echo "  -h, --help             Display this help and exit"
-    echo "  -n, --num-keepers NUM  Set the number of keeper processes"
-    echo "  -w, --work-dir DIR     Set the working directory"
-    echo "  -i, --install          Install all components"
-    echo "  -r, --reset            Reset all components"
-    echo "  -k, --kill             Kill all components"
-    echo "  -s, --stop             Stop all components"
+    echo "  -h|--help       Display this help and exit"
+
+    echo "  -d|--deploy     Start ChronoLog Deployment (default: false)"
+    echo "  -s|--stop       Stop ChronoLog Deployment (default: false)"
+    echo "  -r|--reset      Reset/CleanUp ChronoLog Deployment (default: false)"
+    echo "  -k|--kill       Terminate ChronoLog Deployment (default: false)"
+
+    echo "  -w|--work-dir DIR     Set the working directory"
+    echo "  -u|--output_dir OUTPUT_DIR (default: work_dir/output)"
+    echo "  -v|--visor VISOR_BIN (default: work_dir/bin/chronovisor_server)"
+    echo "  -g|--grapher GRAPHER_BIN (default: work_dir/bin/chrono_grapher)"
+    echo "  -p|--keeper KEEPER_BIN (default: work_dir/bin/chrono_keeper)"
+    echo "  -c|--client CLIENT_BIN (default: work_dir/bin/client_lib_multi_storytellers)"
+    echo "  -f|--conf_file CONF_FILE (default: work_dir/conf/default_conf.json)"
+
+    echo "  -n|--keepers-group  Set the number of keeper processes per group"
```

Comment:
the name `keepers-group` is misleading. since Inna wants to have uneven recording group sizes, we'd better to ask user for total number of keepers.

Reply:
This is the latest description Inna wrote about this: 
_(# 165 ) high priority Local deployment script needs to be cleaned up  to check for missing things at every step instead of proceeding silently. Also the logic of Recording Group to Keeper mappings needs to be more straightforward. The parameters should be # of Recording groups and # of Keepers per group_

I based on that description to make that change. If requested I can change it again.

Reply:
Kun is correct.  I contradict myself with the description I put in this issue.  
So here' the corrected version:
When we have HPC deployment it's more realistic to have a number of grapher hosts and a number of keeper hosts available for the deployment. 
So the logic I suggest for both local and HPC deployment scripts is to start with the number of graphers, create config files for the graphers.  Then start assigning the keepers to graphers iterating through the graphers created, continue until we run out of the keepers.